plugin/trino-faker/src/main/java/io/trino/plugin/faker/ColumnInfo.java

```diff
@@ -35,6 +36,11 @@ public String name()
         return metadata.getName();
     }
 
+    public Type type()
```

Comment:
Remove an unsued method.

Reply:
I've left this here by mistake. This PR is extracted from #24098. I'm working on an alternative version, where I'll use it, so if you don't mind, can we delay removing it for a few days? If it'll take me longer to prepare the next PR, I'll remove it.

Reply:
Got it. The approach looks good to me. Thanks for explaining the context!